AWS/Lambda/CommitEmailer/commit_emailer.py

```diff
@@ -175,8 +175,13 @@ def lambda_handler(event, context):
             # mail_to = os.environ['MAIL_TO']
             mail_to = project_path_email[project]
 
-            exclude_branches = ['main', 'master']
-            if not any(x in event['ref'] for x in exclude_branches):
+            # Never send mail about personal branches
+            if event['ref'].startswith('refs/heads/users/'):
+                break
```

Comment:
This check should be moved to the beginning of the function and should cause an early exit with a status code like we do here: https://github.com/llvm/llvm-admin/blob/main/AWS/Lambda/PREmailer/pr_emailer.py#L268